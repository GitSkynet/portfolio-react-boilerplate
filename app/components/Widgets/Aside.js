import styled from 'styled-components';

const Aside = styled.aside`
  align-self: flex-start;
  display: flex;
  margin-left: auto;
  background-color: #fff;
  color: #575756;

  > aside:not(:last-of-type) {
    border-right: 1px solid #575756;
  }

  @media (min-width: 480px) {
    padding: 0 1em 0 4em;
    background-color: transparent;
    background-image: linear-gradient(45deg, transparent 17%, #fff 17%);
    background-position-x: 0.5em;
  }
`;

export default Aside;
