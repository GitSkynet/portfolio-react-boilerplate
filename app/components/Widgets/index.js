import React from 'react';

import Aside from './Aside';

import SocialLinks from '../SocialLinks';
import LocaleToggle from '../../containers/LocaleToggle';

function Widgets() {
  return (
    <Aside>
      <LocaleToggle />
      <SocialLinks />
    </Aside>
  );
}

export default Widgets;
