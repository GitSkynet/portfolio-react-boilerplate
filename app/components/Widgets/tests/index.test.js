import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Widgets from '../index';

describe('<Widgets>', () => {
  it('Should render and match the snapshot', () => {
    const wrapper = shallow(<Widgets />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
