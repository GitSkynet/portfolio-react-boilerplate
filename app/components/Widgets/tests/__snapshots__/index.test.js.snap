// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Widgets> Should render and match the snapshot 1`] = `
<Aside__Aside>
  <Connect(LocaleToggle) />
  <SocialLinks />
</Aside__Aside>
`;
